src/routes/api/organisation-routes.ts

```diff
@@ -0,0 +1,69 @@
+import express from 'express';
+export const router = express.Router();
+import { check, oneOf } from 'express-validator';
+import { advancedResults, protect, checkInputError } from '../../middleware';
+import {
+    ALPHA_WHITESPACE_REGEX,
+    INVALID_URL_MSG,
+    INVALID_PHONE_NUMBER_MSG,
+    INVALID_EMAIL_MSG,
+    INVALID_ALPHA_SPACE_MSG,
+    NO_FIELD_UPDATED_MSG,
+} from '../../utils';
+
+// Import organisation controllers
+import {
+    getOrganisations,
+    getOrganisation,
+    createOrganisation,
+    updateOrganisation,
+    deleteOrganisation
+} from '../../controllers/organisations';
+
+// Define input validation
+const validateCreateOrganisationFields = [
+    check('name', INVALID_ALPHA_SPACE_MSG('name')).trim().notEmpty().matches(ALPHA_WHITESPACE_REGEX),
+    check('type', INVALID_ALPHA_SPACE_MSG('type')).optional().trim().matches(ALPHA_WHITESPACE_REGEX),
+    check('about', INVALID_ALPHA_SPACE_MSG('about')).trim().notEmpty().matches(ALPHA_WHITESPACE_REGEX),
```

Comment:
I think the `matches(ALPHA_WHITESPACE_REGEX)` can be removed for now, since this does not affect backend logic, we can be lenient with validation and leave it to frontend.

Same for update route below